src/gameapi.js

```diff
@@ -0,0 +1,32 @@
+const gameScoreUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/0VAg54chbZRX31kPbycC/scores';
```

Comment:
- [ ] Currently, your page is not working, I see that you are calling the API but the page do not display any information that I submit as you can see in the gif below, please fix this issue 🤔

  ![leaderboard](https://user-images.githubusercontent.com/60789317/141342083-77f7bd43-ab37-45f3-9707-83adb2396506.gif)

Reply:
Thanks, Andres for the comments. I actually changed a few things before submitting so you made me notice the issues.

Reply:
- [ ] Nice job so far, but there is still a small issue, when i start the app for the first time I'm not able to see any o the scores, only when i hit the refresh button the records are displayed, please fix this 😄